superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -144,25 +159,30 @@ const RightPane = styled.div`
   overflow-y: scroll;
 `;
 
-const AllTagsWrapper = styled.div`
-  ${({ theme }) => `
-    grid-area: tags;
-    margin: ${theme.gridUnit * 4}px ${theme.gridUnit * 2}px 0;
-    input {
-      font-size: ${theme.typography.sizes.s};
-    }
-  `}
-`;
-
-const CategoriesWrapper = styled.div`
+const Wrapper = styled.div`
```

Comment:
`Wrapper` isn't a very descriptive name. I was going to suggest renaming it so that it indicates what it is wrapping, but then I noticed that it lives directly within the `LeftPane`. Is it possible to merge these styles into `LeftPane` and remove the `Wrapper` component?

Reply:
I think it's possible, I'll try it.